main/src/library/Set.flix

```diff
@@ -517,9 +517,18 @@ namespace Set {
     /// Returns `s` as a mutable set.
     ///
     @Time(1) @Space(1)
-    pub def toMutSet(s: Set[a]): MutSet[a] & Impure =
+    pub def toMutSet(s: Set[a]): MutSet[a, Impure] & Impure =
```

Comment:
Does anything depend on this? Can we remove it?

Reply:
I'll look into it